blog/2021-08-16-state_class_total.md

```diff
@@ -81,5 +46,5 @@ Example of `STATE_CLASS_TOTAL_INCREASING` where the sensor does not reset to 0:
 | :--------------------- | -----: | ---: |
 |   2021-08-01T13:00:00  |  1000  |   0  |
 |   2021-08-01T14:00:00  |  1010  |  10  |
-|   2021-08-01T15:00:00  |     5  |  10  |
-|   2021-08-01T16:00:00  |     10 |  15  |
+|   2021-08-01T15:00:00  |     5  |  15  |
+|   2021-08-01T16:00:00  |    10  |  20  |
```

Comment:
```suggestion
|   2021-08-01T16:00:00  |    10  |  20  |

This state class used to be represented by state class `measurement` in combination with a `last_reset` value. This approach has been deprecated and will be interpreted as a `total_increasing` state class instead with an automatic last reset.
```

Reply:
I slightly tweaked the suggestion before committing it.